content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,141 @@
+
+- :guilabel:`Location`: the location where the product is currently stored in the new warehouse.
+  This can be set as the overall warehouse, or a location within the warehouse.
+- :guilabel:`Product`: the product being added to inventory.
 - :guilabel:`Lot/Serial Number`: the lot that the product belongs to or the serial number used to
-  identify it
-- :guilabel:`On Hand`: the total quantity of the product stored in the location for which inventory
-  is being adjusted; this should be zero for a new location or warehouse
-- :guilabel:`Counted`: the amount of the product that is being added to inventory
-- :guilabel:`Difference`: the difference between the *On Hand* and *Counted* values; this will
-  automatically update to reflect the value entered in the :guilabel:`Counted` column
-- :guilabel:`UoM`: the unit of measure used for counting the product
-
-.. image:: create_a_second_warehouse/product-line-configuration.png
+  identify it, if the product is tracked.
+- :guilabel:`On Hand Quantity`: the total quantity of the product stored in the location for which
+  inventory is being adjusted. For a new warehouse or location, this should be set to `0.00`.
+- :guilabel:`UoM`: the unit of measure (UoM) used for counting the product.
+- :guilabel:`Counted Quantity`: the amount of the product being added to inventory.
+- :guilabel:`Difference`: the difference between the on-hand and counted quantities. This
+  automatically updates to reflect the value entered in the :guilabel:`Counted Quantity` column.
+- :guilabel:`Scheduled Date`: the date selected for the next inventory count for this product.
+- :guilabel:`User`: the user who recorded the inventory adjustment in the database.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-inventory-adjustment.png
    :align: center
-   :alt: Include a line for each product being added to inventory.
+   :alt: Filled out inventory adjustment line for product in new warehouse.
+
+Once each line for the products being added to the new warehouse is configured, click
+:guilabel:`Apply` on each line to apply the new inventory counts.
+
+The values in the :guilabel:`On Hand Quantity` column update to reflect those in the
+:guilabel:`Counted Quantity` column, and the products added appear in the new warehouse's stock.
+
+.. note::
+   If products tracked using a lot number or serial number are added without specifying the tracking
+   number, a :guilabel:`Tracked Products in Inventory Adjustment` pop-up window appears upon
+   clicking :guilabel:`Apply` on that product's line.
+
+   The pop-up indicates tracked products were added, but the lots/serial numbers were not defined.
+   To apply the adjustment without the numbers, click :guilabel:`Apply`. To add the numbers before
+   applying, click :guilabel:`Discard`, and add the numbers. Then, once ready, click
+   :guilabel:`Apply` again.
+
+   .. image:: create_a_second_warehouse/create-a-second-warehouse-popup-window.png
```

Comment:
Consider capturing the full pop-up window here, as only the right and bottom borders are visible.

Reply:
Retook screenshot and replaced with better image with all borders visible